### ISSUE TITLE
have some mistake in the loss computation.

### ISSUE THREAD

**Issue by tantao258** [YunYang1994/tensorflow-yolov3#11]
when compute loss_class:
loss_class = tf.nn.sigmoid_cross_entropy_with_logits(labels=y_true_i[..., 5:], logits=pred_boxes_class)

the pred_boxes_class have computed by sigmoid function, I think here can’t use tf.nn.sigmoid_cross_entropy_with_logits
